pennylane/qnode_new/jacobian.py

```diff
@@ -335,3 +337,53 @@ def _pd_analytic_var(self, idx, args, kwargs, **options):
             array[float]: partial derivative of the node
         """
         raise NotImplementedError
+
+    def to_torch(self):
+        """Attach the Torch interface to the Jacobian QNode.
+
+        Raises:
+            QuantumFunctionError: if PyTorch is not installed
+        """
+        # Placing slow imports here, in case the user does not use the Torch interface
+        try:  # pragma: no cover
+            from pennylane.interfaces.torch import TorchQNode
+        except ImportError:  # pragma: no cover
+            raise QuantumFunctionError(
+                "PyTorch not found. Please install " "PyTorch to enable the 'torch' interface."
+            ) from None
+
+        return TorchQNode(self)
+
+    def to_tf(self):
+        """Attach the TensorFlow interface to the Jacobian QNode.
+
+        Raises:
+            QuantumFunctionError: if TensorFlow >= 1.12 is not installed
+        """
+        # Placing slow imports here, in case the user does not use the TF interface
+        try:  # pragma: no cover
+            from pennylane.interfaces.tf import TFQNode
+        except ImportError:  # pragma: no cover
+            raise QuantumFunctionError(
+                "TensorFlow not found. Please install "
+                "the latest version of TensorFlow to enable the 'tf' interface."
+            ) from None
+
+        return TFQNode(self)
+
+    def to_autograd(self):
+        """Attach the TensorFlow interface to the Jacobian QNode.
+
+        Raises:
+            QuantumFunctionError: if TensorFlow >= 1.12 is not installed
+        """
+        # Placing slow imports here, in case the user does not use the TF interface
+        try:  # pragma: no cover
+            from pennylane.interfaces.autograd import to_autograd
```

Comment:
I would rename the function while importing, it is weird that it has the same name as the surrounding function.
```python
from pennylane.interfaces.autograd import to_autograd as _to_autograd
```